examples/02_decoding/plot_haxby_glm_decoding.py

```diff
@@ -102,7 +103,7 @@
     # grab the fmri data for that particular session
     fmri_session = index_img(func_filename, sessions == session)
 
-    # fit the glm
+    # fit the GLM
```

Comment:
```suggestion
    # fit the :term:`GLM
```

Reply:
This is in a code block so we shouldn't link it but could link the glm term in the section header